src/views/tutorial-view/components/next-previous/components/directional-link-box/index.tsx

```diff
@@ -22,23 +22,33 @@ const IconDict: { [k in DirectionOption]: typeof IconArrowRight16 } = {
 function DirectionalLinkBox({
 	href,
 	label,
+	name,
 	ariaLabel,
 	direction,
 }: DirectionalLinkBoxProps) {
 	const Icon = IconDict[direction]
 
 	return (
-		<CardLink
+		<Link
 			className={classNames(s.linkbox, s[`direction-${direction}`])}
 			href={href}
-			ariaLabel={ariaLabel}
+			aria-label={ariaLabel}
 			data-heap-track={`directional-link-box-${direction}`}
 		>
-			<Icon className={classNames(s.icon, s[`direction-${direction}`])} />
-			<Text className={s.text} asElement="span" size={200} weight="medium">
-				{label}
-			</Text>
-		</CardLink>
+			<span className={s.directionLabel}>
+				{' '}
+				<Icon className={classNames(s.icon, s[`direction-${direction}`])} />
+				<Text
+					className={s.labelText}
+					asElement="span"
+					size={200}
+					weight="medium"
+				>
+					{label}
+				</Text>
+			</span>
+			<Text className={s.name}>{name}</Text>
```

Comment:
⛏️ Nitpick: since `name` is optional, I think we might want to do like:

```suggestion
			{name ? <Text className={s.name}>{name}</Text> : null}
```

Reply:
oops, good call!